docs/syntax_and_semantics/exception_handling.md

```diff
@@ -207,6 +207,40 @@ ensure
 end
 ```
 
+## Suffix forms of `rescue` and `ensure`
+
+You can use the suffix form of `rescue` to create one-liner exception handling.
+
+```crystal
+text = File.read("this_file_may_not_exist") rescue "File not found"
```

Comment:
**polish:** This example is technically fine, but I'm wondering if it's a good idea to demonstrate this kind of error handling, when the result in `text` would be impossible to discern the error case from a file which literally contains the text "File not found".
Maybe it would be better to use `rescue nil` instead? This seems like a reasonable use case: Read the file if it exists, but don't care it it doesn't (or is unreadable), while `nil` clearly expresses the error state.